src/main/java/com/puppycrawl/tools/checkstyle/gui/TreeTable.java

```diff
@@ -184,7 +184,8 @@ public void updateUI() {
                 "Tree.foreground", "Tree.font");
     }
 
-    /* Workaround for BasicTableUI anomaly. Make sure the UI never tries to
+    /**
+     * Workaround for BasicTableUI anomaly. Make sure the UI never tries to
```

Comment:
I dont think this is a valid fix. MissingAsterisk checks only javadocs, but it is not a javadoc comment.
Same for other cases below

Reply:
Done, updated.